.github/workflows/build_scala-2.11_spark-2.4.yml

```diff
@@ -0,0 +1,20 @@
+name: Build for Scala 2.11, Spark 2.4
+
+on:
+  push:
```

Comment:
Why on push? Don't you have the master branch locked against pushes?

Reply:
I want to run builds when a PR is created and then it is merged. Pushed are indeed disabled, but the push event is triggered when a PR is merged.

Also, I want to add several badges (2 now, and 1 will add later). So I split builds for Scala 2.11 and 2.12 into 2 workflows. This is because a badge can only refer to a workflow as far as I understood.

Reply:
I get it. But if you have blocked pushes to master and need a PR to push to master and PR cannot be merged unless it is up to date and checks ran, it might not be needed and might run extra workflows. Not sure about the extra cost coming from that. Might be me just being too nit-picky

Badges, we need to add those to Atum as well.

Reply:
It happened a couple of times when 2 PRs are building fine separately, but once they both merged the build fails.

Reply:
`Require branches to be up to date before merging` this in settings is set to true, so it should not happen. In theory.

Reply:
This is a good point. I'll think about adding this requirement and removing build on push.

P.S. Although I've merged the PR, there are still things I want to do. Adding support for Spark 3 builds, etc.